config/config_test.go

```diff
@@ -169,48 +142,60 @@ func TestGetHostname(t *testing.T) {
 }
 
 func TestDDAgentMultiAPIKeys(t *testing.T) {
-	// if no endpoint is given but api_keys are there, match the first api_key
-	// with the default endpoint
+	config.Datadog = config.NewConfig("datadog", "DD", strings.NewReplacer(".", "_"))
```

Comment:
do we need to do this given that `NewAgentConfig` will set the global `config.Datadog` ? Same comment on all of these tests

Reply:
We really just want it reset after the last test, but is more of a best practice to add it to each test to ensure state is reset.